zombies.js

```diff
@@ -79,6 +90,24 @@
  * @property {method} getMaxHealth         Returns private variable `maxHealth`.
  */
 
+function Player(name, health, strength, speed) {
+  this._pack = [];
```

Comment:
`_pack and _maxHealth` should be private variables by using const instead of this.